### PR TITLE
Use the acceptancetesting transport

### PR DESCRIPTION
**Pull request by timbussmann**
uses the AcceptanceTestingTransport based upon the learning transport to ensure it's using a transport which is making use of the subscription and timeout persistence options.

As MSMQ has the same effect, I think this is mostly a @Particular/ravendb-persistence-maintainers decision whether you'd prefer to use msmq (to potentially test DTC scenarios) or whether you want to use the new acceptance testing transport.

### CODE REVIEW

**Comment by timbussmann** [comment]
ready for review  @Particular/ravendb-persistence-maintainers

**Comment by bording** [comment]
I believe this should be against the release branch, not develop.

**Comment by mauroservienti** [comment]
Thanks @timbussmann, can you retarget this to release-5.0?
Interesting to notice that tests that fail locally don’t fail on TC

**Comment by timbussmann** [comment]
retargeted to the release branch.

**Comment by timbussmann** [comment]
the rebase screwed up the package imports. Pushed a fix.

File: src/NServiceBus.RavenDB.AcceptanceTests/NServiceBus.RavenDB.AcceptanceTests.csproj [hunk 1]
```diff
@@ -10,9 +10,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.*" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0-beta0012" />
     <PackageReference Include="NServiceBus.Gateway.AcceptanceTests.Sources" Version="3.0.0-beta0005" />
-    <PackageReference Include="NServiceBus.Transport.Msmq" Version="1.0.0-beta0006" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0-beta0013" />
```
Comment (bording) [inline src/NServiceBus.RavenDB.AcceptanceTests/NServiceBus.RavenDB.AcceptanceTests.csproj]: nitpick: this line should be above the Gateway package line to be sorted properly

**Pull request merged**